modules/fedimint-dummy-server/src/db.rs

```diff
@@ -62,47 +77,51 @@ pub async fn migrate_to_v1(dbtx: &mut DatabaseTransaction<'_>) -> Result<(), any
 
 /// Lookup tx outputs by key or prefix
 #[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash, Serialize)]
-pub struct DummyOutcomeKeyV1(pub OutPoint);
+pub struct DummyOutcomeKey(pub OutPoint);
 
 #[derive(Debug, Encodable, Decodable)]
-pub struct DummyOutcomeKeyV1Prefix;
+pub struct DummyOutcomePrefix;
 
 impl_db_record!(
-    key = DummyOutcomeKeyV1,
+    key = DummyOutcomeKey,
     value = DummyOutputOutcome,
     db_prefix = DbKeyPrefix::Outcome,
 );
-impl_db_lookup!(
-    key = DummyOutcomeKeyV1,
-    query_prefix = DummyOutcomeKeyV1Prefix
-);
+impl_db_lookup!(key = DummyOutcomeKey, query_prefix = DummyOutcomePrefix);
 
-/// Lookup funds for a user by key or prefix
+/// Lookup signature requests by key or prefix
 #[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash, Serialize)]
-pub struct DummyFundsKeyV1(pub XOnlyPublicKey);
+pub struct DummySignatureShareKey(pub String, pub PeerId);
 
-#[derive(Debug, Encodable, Decodable)]
-pub struct DummyFundsKeyV1Prefix;
+#[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash, Serialize)]
+pub struct DummySignatureShareStringPrefix(pub String);
+
+#[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash, Serialize)]
+pub struct DummySignatureSharePrefix;
 
 impl_db_record!(
-    key = DummyFundsKeyV1,
-    value = Amount,
-    db_prefix = DbKeyPrefix::Funds,
+    key = DummySignatureShareKey,
+    value = SerdeSignatureShare,
+    db_prefix = DbKeyPrefix::SignatureShare,
+);
+impl_db_lookup!(
+    key = DummySignatureShareKey,
+    query_prefix = DummySignatureShareStringPrefix,
+    query_prefix = DummySignatureSharePrefix
 );
-impl_db_lookup!(key = DummyFundsKeyV1, query_prefix = DummyFundsKeyV1Prefix);
 
 /// Lookup signature requests by key or prefix
 #[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash, Serialize)]
-pub struct DummySignKeyV1(pub String);
+pub struct DummySignatureKey(pub String);
 
 #[derive(Debug, Encodable, Decodable)]
-pub struct DummySignV1Prefix;
+pub struct DummySignaturePrefix;
```

Comment:
Why the rename? I'm not too hung up about it, but pinging @jkitman. Generally renaming is nicer as a separate commit/PR since it can be done fully automatically and just adds noise.